index.html

```diff
@@ -1,76 +1,105 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0"> 
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <link rel="icon" type="image/png" sizes="32x32" href="./images/favicon-32x32.png">
   <title>Frontend Mentor | Sunnyside agency landing page</title>
   <link rel="preconnect" href="https://fonts.googleapis.com">
   <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-  <link href="https://fonts.googleapis.com/css2?family=Barlow:wght@600&family=Fraunces:opsz,wght@9..144,700;9..144,900&display=swap" rel="stylesheet">
+  <link
+    href="https://fonts.googleapis.com/css2?family=Barlow:wght@600&family=Fraunces:opsz,wght@9..144,700;9..144,900&display=swap"
+    rel="stylesheet">
   <link rel="stylesheet" href="./styles/main.css">
 </head>
-<body>
-  About
-  Services
-  Projects
-  Contact
-
-  We are creatives
-
-<main class="body__container">
-  <section class="body__container--articles">
 
-    <div class="articles__container">
-      <div class="articles__container--text">
-        <h2 class="text__header">Transform your brand</h2>
-        <p class="text__content">
-          We are a full-service creative agency specializing in helping brands grow fast. 
-          Engage your clients through compelling visuals that do most of the marketing for you.
-        </p>
-        <a href="#" class="text__link">Learn more</a>
+<body>
+  <nav class="bg__color--blue">
+    <div class="body__container--nav">
+      <p class="nav__name">sunnyside</p>
```

Comment:
@Jelena-ag to z założenia byłoby logo, więc raczej zrobiłabym je klikalne.

Reply:
zrobione: 
`  <a class="nav__name" href="#">sunnyside</a>`